daemon/access.go

```diff
@@ -30,21 +30,12 @@ import (
 	"github.com/snapcore/snapd/polkit"
 )
 
-type accessResult int
-
-const (
-	accessOK accessResult = iota
-	accessUnauthorized
-	accessForbidden
-	accessCancelled
-)
-
 var (
 	polkitCheckAuthorization = polkit.CheckAuthorization
 	checkPolkitAction        = checkPolkitActionImpl
 )
 
-func checkPolkitActionImpl(r *http.Request, ucred *ucrednet, action string) accessResult {
+func checkPolkitActionImpl(r *http.Request, ucred *ucrednet, action string) Response {
```

Comment:
I think this would be clearer/more readable if we had something like APIError (as an interface parallel or derived from Response),  that would be its own separate refactoring though, I need to think about it. Maybe we should have at least a TODO here.

Reply:
We've now got apiError, so I've updated this code to match.